packages/abstract-signer/src.ts/index.ts

```diff
@@ -186,13 +187,12 @@ export abstract class Signer {
         } else {
             hederaTx.setContractId(to);
         }
-
-        // TODO: the exact amount here will be computed using getCost when it's implemented
-        const cost = 3;
+        const gasLimit = BigNumber.from(tx.gasLimit).toNumber();
+        const cost = DEFAULT_HEDERA_CALL_TX_FEE * 10 + gasLimit * CALL_GAS_PRICE_TINYBARS;
```

Comment:
nit: I think that it would be better to extract the `10` in a const above  and state that this is a buffer for fluctuating prices. We can multiple it by `20` as-well. Just to be sure that we won't brake someone in the case that 6 months from now, hbars have dropped in value or the user provides many `signatures` as part of his transaction (meaning that more will be charged for sig verification)